### DISCUSSION TITLE
part-ii/2017-0/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#386] [Question comments]
Part II 2017 Algebraic Geometry: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2017-0/

**Comment by artie2000** [comment]
(a),(b) are bookwork.

(c): There are no singular points.
<details>
<summary>Characteristic 2:</summary>

Every point is singular as $\nabla f=0$.

</details>
